### PR TITLE
Change price and how pdf download works

### PR DESCRIPTION
**Pull request by fellnerse**
I would suggest to lower the price to something no one thinks about if its too much: 0.99€
also they way stuff is presented is a bit changed

### CODE REVIEW

File: components/DownloadPopup.vue [hunk 1]
```diff
@@ -82,6 +83,17 @@
...
         </v-dialog>
+
+        <div class="text-text-h3 my-4">
+          Looking for <b>PDF download</b> instead?
+        </div>
+        <v-btn
+          color="#07bc4c"
+          dark
+          @click="$vuetify.goTo('#payButton', { duration: 300, offset: 100 })"
+          ><v-icon class="mr-2">mdi-keyboard-return</v-icon>go to PDF
```
Comment (PKehnel) [inline components/DownloadPopup.vue:94]: a consistent design with the other goto buttons would be cool. 
Maybe the reverse Icon of the download button? 

save2
Save1

File: components/ChatVisualization/ChatVisualization.vue [hunk 2]
```diff
@@ -107,51 +97,52 @@ export default {
       showDownloadPopup: false,
       ego: this.chat.messagesPerPerson[0].name,
       isLoading: false,
+      price: 0.99,
+      currency: "EUR",
```
Comment (PKehnel) [inline components/ChatVisualization/ChatVisualization.vue:101]: If we post on data is beautiful, maybe $ is better? As the main user group from Reddit are Americans, https://www.statista.com/statistics/325144/reddit-global-active-user-distribution/

**Review by PKehnel — CHANGES_REQUESTED** [review]

**Comment by PKehnel** [comment]
And for the price I'm fine with 1$ / 1€ for now, would be good to do a "price analysis" next week. (I saw the reddit comment)

File: components/ChatVisualization/ChatVisualization.vue (hunk 2 above)
Comment (mowolf) [reply components/ChatVisualization/ChatVisualization.vue:101]: Nah, We need to switch currencies in EUR and then we need to take the rather low PayPal currency exchange rates.

File: components/DownloadPopup.vue (hunk 1 above)
Comment (mowolf) [reply components/DownloadPopup.vue:94]: I think that's fine

File: components/ChatVisualization/ChatVisualization.vue [hunk 3]
```diff
@@ -107,51 +97,52 @@ export default {
...
     },
     onApprove(event) {
-      render(this.chat, this.attachments, this.ego, false);
       console.log("approved", event);
+      this.download();
     },
     onError(event) {
-      console.log("approved", event);
-
       console.log("error", event);
```
[the remark this answers is not in the document]
Comment (mowolf) [reply components/ChatVisualization/ChatVisualization.vue:127]: might be helpful for debugging payment

File: components/ChatVisualization/ChatVisualization.vue [hunk 4]
```diff
@@ -107,51 +97,52 @@ export default {
...
         event_category: "download",
         event_label: "download-pdf",
         value: "10",
       });
       this.isLoading = true;
-      let done = await render(this.chat, this.attachments, this.ego, false);
-      if (done) {
-        this.isLoading = false;
-      }
+      render(this.chat, this.attachments, this.ego, false).then(
```
[the remark this answers is not in the document]
Comment (mowolf) [reply components/ChatVisualization/ChatVisualization.vue:115]: the gtag is different and this is just calling two functions, I feel this is simple enough

**Review by mowolf — APPROVED** [review]
I like it :)

I see Pauls Points but I would merge it the way it is.

File: components/DownloadPopup.vue (hunk 1 above)
Comment (fellnerse) [reply components/DownloadPopup.vue:94]: That's not part of this PR.
We should have Standart btns somewhere, but for now everywhere this random hexcode and other params are used

File: components/ChatVisualization/ChatVisualization.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (fellnerse) [reply components/ChatVisualization/ChatVisualization.vue:127]: You can suggest changes which would be easier
```suggestion
```

File: components/ChatVisualization/ChatVisualization.vue [hunk 5]
```diff
@@ -107,51 +97,52 @@ export default {
...
+      );
     },
     onCreateOrder(data, actions) {
-      console.log("approved", event);
-
       console.log("order created", data, actions);
     },
     onApprove(event) {
-      render(this.chat, this.attachments, this.ego, false);
       console.log("approved", event);
```
[the remark this answers is not in the document]
Comment (fellnerse) [reply components/ChatVisualization/ChatVisualization.vue:123]:
```suggestion
```

**Pull request merged**